docker/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex
+
+gcloud config set project istio-test
```

Comment:
OK. This bit just duplicates the docker/build.sh from istio/manager. I'll rename to gcloud_build.sh. Ultimately, we just need a place to put our images.